yarn-project/stdlib/src/config/config.ts

```diff
@@ -20,7 +20,13 @@ export const chainConfigMappings: ConfigMappingsType<ChainConfig> = {
   version: {
     env: 'VERSION',
     description: 'The version of the rollup.',
-    parseEnv: (val: string) => (Number.isSafeInteger(parseInt(val, 10)) ? parseInt(val, 10) : undefined),
+    parseEnv: (val: string) => {
+      const parsed = parseInt(val, 10);
```

Comment:
Prior to parse int, lets check for non numeric characters, and lets also add that to the number helper